packages/netstring/rollup.config.js

```diff
@@ -0,0 +1,104 @@
+import resolve from '@rollup/plugin-node-resolve';
```

Comment:
is this rollup config copied from somewhere? i.e. has it already been extensively reviewed?

Reply:
Yes, our rollup configurations are copied by scaffolding from `/etc/rollup.config.js` and this is used everywhere. This particular rollup however is modified to provide separate reader and writer entries to allow one or the other to be left out of bundles. The rollup config, package.json, and top-level .js files conspire to provide external entry-points for each of the properjs packages, with varying strategies depending on what environment is loading the file. Rollup provides a backward-compatibility story for CommonJS environments. Top level `./index.js`, `./reader.js`, and `./writer.js` provide entry points for old versions of Node.js, those predating the addition of the `exports` property. This is an actively evolving story as we figure out how to provide sufficiently broad compatibility. I was tracking here https://github.com/endojs/endo/issues/238 but this is a little stale.